clowdapp.yaml

```diff
@@ -0,0 +1,167 @@
+          httpGet:
+            path: /mgmt/v0/status
+            port: 8080
+            initialDelaySeconds: 10
+            periodSeconds: 10
+        resources:
+          limits:
+            cpu: ${CPU_LIMIT}
+            memory: {MEMORY_LIMIT}
+          requests:
+            cpu: ${CPU_REQUEST}
+            memory: ${MEMORY_REQUEST}
+        volumes:
+          - emptyDir: {}
+            name: drift-prometheus-data
+        volumeMounts:
+          - mountPath: /prometheus-data
+            name: drift-prometheus-data
+        env:
+          - name: APP_CONFIG
+            value: gunicorn.conf.py
+          - name: BASELINE_SVC_URL
+            value: "http://${SYSTEM_BASELINE_HOST}:8080"
+          - name: HSP_SVC_URL
+            value: "http://${HSP_HOST}:8080"
+          - name: RBAC_SVC_URL
+            value: "http://${RBAC_HOST}:8080"
+          - name: ENABLE_RBAC
+            value: "${DRIFT_ENABLE_RBAC}"
+          - name: ENABLE_SMART_MANGEMENT_ENTITLEMENT_CHECK
+            value: "${ENABLE_SMART_MANGEMENT_ENTITLEMENT_CHECK}"
+          - name: WEB_CONCURRENCY
+            value: "4"
+          - name: PATH_PREFIX
+            value: /api/
+          - name: APP_NAME
+            value: drift
+          - name: ENABLE_PIPENV
+            value: 'true'
+          - name: prometheus_multiproc_dir
+            value: /prometheus-data
+          - name: INVENTORY_SVC_URL
+            value: "http://${HOST_INVENTORY_HOST}:8080"
+          - name: CW_LOG_GROUP
+            value: "${CLOUD_WATCH_LOG_GROUP}"
+          - name: AWS_REGION
+            value: 'us-east-1'
+          - name: CW_AWS_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                key: CW_AWS_ACCESS_KEY_ID
+                name: cloudwatch
+          - name: CW_AWS_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                key: CW_AWS_SECRET_ACCESS_KEY
+                name: cloudwatch
+          - name: DRIFT_SHARED_SECRET
+            valueFrom:
+              secretKeyRef:
+                key: shared-secret
+                name: drift-interpod-comms
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: drift-backend
+    name: drift-backend
+    annotations:
+      prometheus.io/path: /mgmt/v0/metrics
+      prometheus.io/port: '8080'
+      prometheus.io/scrape: 'true'
+  spec:
+    type: ClusterIP
+    selector:
+      deploymentconfig: drift-backend
+    ports:
+      - name: 8080-tcp
+        port: 8080
+        protocol: TCP
+        targetPort: 8080
+
+- apiVersion: autoscaling/v1
+  kind: HorizontalPodAutoscaler
+  metadata:
+    name: drift-backend
+    labels:
+      app: drift-backend
+  spec:
+    scaleTargetRef:
+      apiVersion: v1
+      kind: DeploymentConfig
+      name: drift-backend
+    minReplicas: ${{MIN_REPLICAS}}
+    maxReplicas: ${{MAX_REPLICAS}}
+    targetCPUUtilizationPercentage: 50
+
+parameters:
+  - name: HOST_INVENTORY_HOST
```

Comment:
The dependent services like inventory will be provided by clowder and need to be set at runtime reading the values using the Clowder config helper lib

Reply:
the same goes for rbac , etc.